### ISSUE TITLE
issue about learning curve

### ISSUE THREAD

**Issue by xujiayuan-tj** [selcukorkmaz/fastml#173]
**Dear Selcuk,
I am trying to implement a learning curve with a simple regression test, but it keeps crashing. Could you help me check if the issue lies with my dataset or the way I am implementing the code?"**

-------------------------------------------------------------------------------------
> fastml_fit <- do.call(fastml, fastml_params)
Splitting data into training and test sets...
Creating preprocessing recipe...
Training models: rand_forest, xgboost, svm_rbf, linear_reg
Evaluating models...
Best model selected: xgboost (xgboost)
Preparing preprocessing recipe for downstream use...
Learning curve: using 10% of training data
→ A | warning: A correlation computation is required, but `estimate` is constant and has 0 standard deviation, resulting in a divide
               by 0 error. `NA` will be returned.
→ A | warning: A correlation computation is required, but `estimate` is constant and has 0 standard deviation, resulting in a divide
               by 0 error. `NA` will be returned.
→ A | warning: A correlation computation is required, but `estimate` is constant and has 0 standard deviation, resulting in a divide
               by 0 error. `NA` will be returned.
→ A | warning: A correlation computation is required, but `estimate` is constant and has 0 standard deviation, resulting in a divide
               by 0 error. `NA` will be returned.
→ A | warning: A correlation computation is required, but `estimate` is constant and has 0 standard deviation, resulting in a divide
               by 0 error. `NA` will be returned.
→ A | warning: A correlation computation is required, but `estimate` is constant and has 0 standard deviation, resulting in a divide
               by 0 error. `NA` will be returned.
→ A | warning: A correlation computation is required, but `estimate` is constant and has 0 standard deviation, resulting in a divide
               by 0 error. `NA` will be returned.
→ A | warning: A correlation computation is required, but `estimate` is constant and has 0 standard deviation, resulting in a divide
               by 0 error. `NA` will be returned.
→ A | warning: A correlation computation is required, but `estimate` is constant and has 0 standard deviation, resulting in a divide
               by 0 error. `NA` will be returned.
→ A | warning: A correlation computation is required, but `estimate` is constant and has 0 standard deviation, resulting in a divide
               by 0 error. `NA` will be returned.
warning:
1: In (function (data = NULL, train_data = NULL, test_data = NULL,  :
  Missing values in numeric predictors are being imputed using the median.
2: In for (i in seq_len(...length())) { :
  (<-kubernetes.docker.internal:11170)
-------------------------------------------------------------------------------------
**NEXT is my parameters**

> fastml_params
$data
# A tibble: 48,026 × 4
     bmi shen_gao ti_zhong nian_ling
   <dbl>    <dbl>    <dbl>     <dbl>
 1  36.5     190     120.         54
 2  25.8     185.     88.6        60
 3  25.6     175      78.5        61
 4  32.2     170      93.1        55
 5  29.4     173      87.9        44
 6  21.8     173      65.3        68
 7  28.5     177      89.2        69
 8  23.3     165      63.3        51
 9  26.7     176      82.7        65
10  27.1     154      64.3        45
# ℹ 48,016 more rows
# ℹ Use `print(n = ...)` to see more rows

$label
[1] "bmi"

$task
[1] "regression"

$algorithms
[1] "rand_forest" "xgboost"     "svm_rbf"     "linear_reg" 

$test_size
[1] 0.3

$stratify
[1] TRUE

$resampling_method
[1] "cv"

$folds
[1] 10

$use_default_tuning
[1] TRUE

$tuning_strategy
[1] "grid"

$tuning_iterations
[1] 10

$metric
[1] "rmse"

$bootstrap_ci
[1] TRUE

$bootstrap_samples
[1] 500

$learning_curve
[1] TRUE

$n_cores
[1] 12

$verbose
[1] TRUE

$seed
[1] 123

$impute_method
[1] "medianImpute"

$scaling_methods
[1] "center" "scale"
